Makefile

```diff
@@ -1,16 +1,34 @@
-all: env test packages
+all: env packages doc test
+all-docker: env-docker packages-docker doc-docker test-docker
 
 clean:
-	bin/clean.sh
+	@bash bin/clean.sh
+
+distclean: clean
```

Comment:
Not new in this PR but these targets appear to be unrelated to the names of files that they create and as such they should be [`PHONY`](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) targets.